### PR TITLE
fix(incels.is): NSFW fix

### PR DESCRIPTION
**Pull request by ION606**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by ION606** [comment]
that was so dumb

File: websites/C/Character.AI/metadata.json
Comment (theusaf) [inline websites/C/Character.AI/metadata.json:1]: Make sure to revert changes to files not part of this PR.

**Review by theusaf — CHANGES_REQUESTED** [review]
